clusters/l3-sqnc/cert-manager/release.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: cert-manager 
+  namespace: cert-manager
+spec:
+  install:
+    remediation:
+      retries: -1
+  releaseName: cert-manager
+  chart:
+    spec:
+      chart: cert-manager  
+      sourceRef:
+        kind: HelmRepository
+        name: bitnami
+  interval: 10m0s
+  # Default values
+  # https://github.com/bitnami/charts/blob/master/bitnami/cert-manager/values.yaml
+  valuesFrom:
+    - kind: ConfigMap
+      name: cert-manager-values
+      valuesKey: values-cert-manager.yaml
```

Comment:
Sort out line end here.

Reply:
went through entire folder to make sure formatting is consistent across